README.md

```diff
@@ -7,6 +7,11 @@ Setup is simple. Simply run:
 rake db:migrate
 ```
 
+To run the app, type in your console:
+```bash
+rails s
+```
+
```

Comment:
Instead of just saying this, how about also add some text like: "While the server is running, trying poking around the app. Notice there is no way to mark if a task is done. That is the feature we are going to implement."